pkg/sloop/storemanager/storemanager.go

```diff
@@ -158,42 +165,52 @@ func (sm *StoreManager) refreshStats() *storeStats {
 	return sm.stats
 }
 
-func doCleanup(tables typed.Tables, timeLimit time.Duration, sizeLimitBytes int, stats *storeStats) (bool, error) {
+func doCleanup(tables typed.Tables, timeLimit time.Duration, sizeLimitBytes int, stats *storeStats) (bool, float64, error) {
 	ok, minPartition, maxPartition, err := tables.GetMinAndMaxPartition()
 	if err != nil {
-		return false, fmt.Errorf("failed to get min partition : %s, max partition: %s, err:%v", minPartition, maxPartition, err)
+		return false, 0, fmt.Errorf("failed to get min partition : %s, max partition: %s, err:%v", minPartition, maxPartition, err)
 	}
 	if !ok {
-		return false, nil
+		return false, 0, nil
 	}
 
+	var totalNumOfDeletedKeys float64 = 0
+	var minPartitionAge float64 = 0
```

Comment:
scope of `minPartitionAge` seems to large - if moved to smaller scope, `var` statement can be eliminated